site/.vitepress/sidebar/index.ts

```diff
@@ -78,11 +78,11 @@ export const sidebar: DefaultTheme.Sidebar = [
         collapsed: true,
         items: [
           {
-            text: "Using Your Own",
+            text: "Use your own",
             link: "/guides/custom-signer",
           },
           {
-            text: "Contributing Your Signer",
+            text: "Contribute your Signer",
```

Comment:
```suggestion
            text: "Contribute your signer",
```

Reply:
same here, signer is a proper noun like Smart Account